app/celery/reporting_tasks.py

```diff
@@ -103,15 +103,16 @@ def create_nightly_notification_status():
             )
             current_app.logger.info(
                 f"create-nightly-notification-status-for-day task created "
-                f"for type {notification_type} for {process_day}"
+                f"for {notification_type} and {process_day}"
```

Comment:
"task created for sms *for* 12-01-2022" makes a more readable sentence than "task created for sms *and* 12-01-2022"

Reply:
Looking through these again I realise some of them are in different orders as well. In https://github.com/alphagov/notifications-api/pull/3425/commits/9686595fa85c1fc5458f26c3afcc893a8aa93351 I've gone through and made the ordering consistent, so we can have "for" on the end 👍.

Reply:
Nice one 👍🏼